installer/updater/updater.cc

```diff
@@ -37,6 +38,8 @@
 #define MANIFEST_URL_EON_LOCAL "http://192.168.5.1:8000/neosupdate/update.local.json"
 #define MANIFEST_URL_EON "https://github.com/commaai/eon-neos/raw/master/update.json"
 const char *manifest_url = MANIFEST_URL_EON;
+bool background_cache = false;
```

Comment:
why global and used as argument?

Reply:
`run_stages()` is called from two places; only one of them varies that argument based on `background_cache`. The meaning of that argument is distinct from `background_cache`, and `run_stages()` needs to know its actual value either way it's called.

Reply:
I would still pass it around using arguments instead of using a global.